examples/stac-collection-pmtiles-vector.html

```diff
@@ -0,0 +1,11 @@
+---
+layout: example.html
+title: STAC Collection with PMTiles (Vector)
+shortdesc: Rendering a STAC Collection with a selection of web mapping services.
+docs: >
+  Geometry and web map links (PMTiles) from SpatioTemporal Asset Catalog (STAC) Collections can be rendered as layer group.
+  
+  Note: The example doesn't work in Firefox due to restrictive CORS rules at the data host.
```

Comment:
I think we should either:
* mirror the data in this GH repo or in some CORS-friendly bucket for other STAC assets
* I can mirror this at a stable url like r2-public.protomaps.com/stac/foo.pmtiles

Reply:
This would be great, indeed!